frontend/src/components/option/OptionBanner.tsx

```diff
@@ -1,110 +1,141 @@
-                {currentSlice.map((packageOption, idx) => (
-                  <StyledOptionButton
-                    key={startIdx + idx}
-                    isClicked={startIdx + idx === packageIdx}
-                    onClick={handlePackageOption(startIdx + idx)}
-                  >
-                    {packageOption.name}
-                    <StyleDivder length={58} color='gray800' isClicked={startIdx + idx === packageIdx} />
-                  </StyledOptionButton>
-                ))}
-              </Flex>
-              <Icon iconType='ArrowRight' size={24} onClick={nextPage} css={pagingButtonStyle(isEndPage)} />
-            </Flex>
-          )}
-          <Typography font='TextKRRegular12' color='gray800' marginTop={8}>
-            {isPackageOption(optionInfo) ? optionInfo.components[packageIdx].description : optionInfo.description}
-          </Typography>
-          {hasHMGData && isPackageOption(optionInfo) && <HMGDetail {...optionInfo.components[packageIdx].hmgData} />}
-          {hasHMGData && !isPackageOption(optionInfo) && <HMGDetail {...optionInfo.hmgData} />}
-        </MainContainer>
+    <OptionBanner
+      subTitle={optionInfo.category}
+      title={optionInfo.name}
+      imgUrl={optionInfo.components[componentIdx].image}
+    >
+      {/* 패키지 옵션 확인 페이지네이션 */}
+      <Flex gap={8} marginTop={10} alignItems='center'>
+        <Icon iconType='ArrowLeft' size={24} onClick={prevPage} css={pagingButtonStyle(isStartPage)} />
+        <Flex gap={16}>
+          {currentSlice.map((component, idx) => (
+            <StyledOptionButton
+              key={startIdx + idx}
+              isClicked={startIdx + idx === componentIdx}
+              onClick={handleClickComponent(startIdx + idx)}
+            >
+              <span>{component.name}</span>
+              <StyleDivder length={58} color='gray800' isClicked={startIdx + idx === componentIdx} />
+            </StyledOptionButton>
+          ))}
+        </Flex>
+        <Icon iconType='ArrowRight' size={24} onClick={nextPage} css={pagingButtonStyle(isEndPage)} />
       </Flex>
-      <MainImg src={isPackageOption(optionInfo) ? optionInfo.components[packageIdx].image : optionInfo.image} />
-      <StyledImageButton onClick={handleClickButton}>{isCheckImg ? '이미지 접기' : '이미지 확인'}</StyledImageButton>
-      <PriceStaticBar />
-    </StyledOptionBanner>
+      {/* 패키지 옵션 설명 */}
+      <Typography font='TextKRRegular12' color='gray800' marginTop={8} className='description'>
+        {optionInfo.components[componentIdx].description}
+      </Typography>
+      <HMGDetail
+        overHalf={optionInfo.isOverHalf}
+        choiceCount={optionInfo.choiceCount}
+        useCount={optionInfo.components[componentIdx].useCount}
+      />
+      <Typography font='TextKRRegular12' color='gray800' marginTop={8}></Typography>
+    </OptionBanner>
   );
 }
 
+OptionBanner.GeneralOption = GeneralOption;
+OptionBanner.PackageOption = PackageOption;
 export default OptionBanner;
```

Comment:
```suggestion
export default { GeneralOption, PackageOption };
```

OptionBanner 자체를 밖에서 쓸이유가없다면 이런식으로 해도 좋을듯?